src/pages/components/combobox.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Combobox
+description: Combobox is an input plus a list box that appears beneath it. It is used for search and filtering.
+package: '@zendeskgarden/react-dropdowns'
+components:
+  - dropdowns/src/elements/Combobox/Combobox.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
+  - dropdowns/src/elements/Menu/Menu.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To filter down a list of options</li>
+      <li>To quickly find a known option</li>
+      <li>As a search field</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>For a searchable dropdown, use Autocomplete instead</li>
+      <li>To make more than one selection, use Multiselect instead</li>
```

Comment:
Please link "Autocomplete" and "Multiselect" similar to https://garden.zendesk.com/components/autocomplete

Reply:
Updated! Thanks for spotting this